_includes/spawn-sampledatabases.html

```diff
@@ -156,7 +156,7 @@
 
                 <pre
                     class="console"
-                ><span>&gt;</span> spawnctl create data-container --image wide-world-importers:2017 --name spawn-tutorial --lifetime 1h</pre>
+                ><span>&gt;</span> spawnctl create data-container --image mssql-wideworldimporters:v2017 --name spawn-tutorial --lifetime 1h</pre>
```

Comment:
Same comment about the `v` prefix as on https://github.com/red-gate/spawn/pull/4709

Reply:
Going with this